about.hbs

```diff
@@ -18,4 +19,4 @@ module.exports = [
 		],
 	},
 	{{/each}}
-];
+]
```

Comment:
I think we should still have the semicolon. Although perhaps I'm confused what's going on here (see the comment above).

Reply:
if we must nitpick, then `eval()` expects an expression, and an expression does not have a semicolon. But it doesn't matter either way, and it's going to be replaced with proper JSON eventually.